pyscf/adc/radc.py

```diff
@@ -510,6 +655,7 @@ def __init__(self, mf, frozen=0, mo_coeff=None, mo_occ=None):
         self.method = "adc(2)"
         self.method_type = "ip"
         self.with_df = None
+        self.get_mpn_energy = True
```

Comment:
Please rename this to compute_mpn_energy everywhere

Reply:
Done